admin/bootstrap.py

```diff
@@ -64,7 +64,10 @@ def run_command(command: List[str]) -> Iterator[bytes]:
 
 def is_tails() -> bool:
     try:
-        id = subprocess.check_output("lsb_release --id --short", shell=True).decode("utf-8").strip()
+        id = subprocess.check_output(
```

Comment:
Instead of shelling out to awk from Python, can we do the string parsing in Python itself? Something like splitting the lines, check if it starts with "VERSION_CODENAME", then taking the part of it after that and stripping quotes.

Reply:
Yes, makes sense.